Delete_User.php

```diff
@@ -1,56 +1,73 @@
+<?php
+		// Deliver to login page if access to this page with out login
+		session_start();
+		if(!isset($_SESSION['login_user'])){
+				header("location:login.php");
+				}
+		else{
+			$User = $_SESSION['login_user'];
+		}
+?>
+
 <!DOCTYPE html>
 <html lang = "en">
 	<head>
 		<title>Delete User</title>
 		<meta chrset="utf-8"/>
-		<meta name="Keywords" content = "html5, layout, CSS Grid System"/>
+		<meta name="Keywords" content = "html5"/>
 		<meta name = "Author" content = "Tony Nguyen"/>
-		<meta name = "Description" content = "CSS Grid System Layout Tutorial"/>
+		<meta name = "Description" content = "Music database"/>
 		<meta name = "viewport" content = "width=device-width. initial-scale=1"/>
-		
-		<link rel ="stylesheet" href = "Delete_User.css"> 
+		<!--Connected to css -->
+		<link rel ="stylesheet" href = "style.css"> 
 		
 	</head>
 <body>
+	<!--Set for grid system for the page -->
 	<div class ="grid-container">
+		<!-- The div for nav and in the div it contain the logo and a function to called the nav -->
 		<div class="navigation">
+			<!-- Set a div that contain logo for the music database -->
 			<div class="logo">
 				<a href = "01_login_real.php"><img src = "images/logo8.png" height = "100"></a>
 			</div>
 			<?php
-				//Pulls the links from the nav.php page and places them in the navigation div
+				//Pulls the links from the nav_admin.php page and places them in the navigation div
 				require 'nav_admin.php'; //'require' is 100% needed for the site to run 
 			?>
 		</div>
-		<div class="content"><!-- Holds the main page content -->
+		<div class="content"><!-- Holds the delete user form -->
 			<p>Delete User</p>
-			<h3><form method = "post" id = "01_login">
+			<h3><form method = "post">
 			<input id="us" type = "text" name = "Username" placeholder="User name"/><br />
 			<input id="login" type= "submit" value="Delete"/>
 			</form></h3>
 			<?php
+			//Connect.php (tells where to connect servername, username, password, dbaseName)
 				require "musicdatabase_database_mysqli.php";
-				print "<p class = 'a'>Connected to server</p>";
 							
 				if(isset($_POST['Username'])){
 								
 				$UserID = $_POST['Username'];
+				// Delete query use to delelte user account
 				$deletequery = "DELETE FROM User WHERE `User` = '$UserID'";
 
 				if (mysqli_query($conn,$deletequery))
 					{
-					echo "<p class = '#'>Record inserted:</p>";
+					echo "<p class = 'Connected'>Record deleted:</p>";
 					}
 				else
 					{
-					echo "<p class = '#'>Error inserting record:</p>";
+					echo "<p class = 'error'>Error deleting record:</p>";
```

Comment:
I change this so it can display the text that would make more sense and also make it a class for it so the element that I input in css will display in the text.